dotnet/src/Extensions/Experimental/Orchestration.Flow/FlowExecutor/FlowExecutor.cs

```diff
@@ -99,13 +97,13 @@ internal class FlowExecutor : IFlowExecutor
     /// </summary>
     private readonly ISKFunction _checkStartStepFunction;
 
-    internal FlowExecutor(KernelBuilder kernelBuilder, IFlowStatusProvider statusProvider, Dictionary<object, string?> globalSkillCollection, FlowPlannerConfig? config = null)
+    internal FlowExecutor(KernelBuilder kernelBuilder, IFlowStatusProvider statusProvider, Dictionary<object, string?> globalSkillCollection, FlowOrchestratorConfig? config = null)
     {
         this._kernelBuilder = kernelBuilder;
         this._systemKernel = kernelBuilder.Build();
```

Comment:
Why do we name kernel as a system one here?

Reply:
@SergeyMenshykh `_systemKernel` refers to the kernel used by Orchestrator itself.

For each step/action, separate kernels with limited context would be created.
It was named as `_systemKernel` to be consistent with `StepwisePlanner` however that one has been renamed to `_kernel`.
@lemillermicrosoft how do you think?